### PR TITLE
Enter area of parcel used for activitity

### PR DESCRIPTION
**Pull request by fareed-mohammed**
https://eaflood.atlassian.net/browse/SFI-336

### CODE REVIEW

**Comment by davidjamesstone** [comment]
@fareed-mohammed I have update the branch to include the change we just discussed.

Please review and let me know when you would like me to approve.

(BTW, I just noticed the tests (test/integration/narrow/routes/funding-options/land-primary-actions.test.js) might need changing tp include the underscore in the name and drop the hidden data)

File: app/routes/funding-options/land-primary-actions.js [hunk 1]
```diff
@@ -1,32 +1,90 @@
...
+        const { parcels } = await getParcels(agreement.sbi)
+        const landInHectares = getLandInHectares(payload, parcels)
+        const hasInvalidValues = landInHectares.some(element => !element.valid)
+
+        if (hasInvalidValues) {
+          return h.view('funding-options/land-primary-actions', {
+            landInHectares: landInHectares,
+            landParcels: parcels,
+            errors: true
+          }).code(400).takeover()
```
Comment (davidjamesstone) [inline app/routes/funding-options/land-primary-actions.js:55]: Fareed, I'll add a separate ticket to JIRA to discuss returning 400 status codes for HTML forms with errors. Personally, I don't think we should be doing this - it should be 200. I can see it is in other places though so we'll address them all together.

**Review by davidjamesstone — APPROVED** [review]

File: app/routes/funding-options/land-primary-actions.js (hunk 1 above)
Comment (fareed-mohammed) [reply app/routes/funding-options/land-primary-actions.js:55]: Cool, no worries

**Comment by fareed-mohammed** [comment]
Copied the code to a newly created PR branch - https://github.com/DEFRA/ffc-sfi-apply-web/pull/40. As this PR had commits which weren't verified and it was easier to copy the changes into the newly created branch.

**Pull request closed without merging**